indico/modules/events/tracks/controllers.py

```diff
@@ -63,3 +68,39 @@ def _process(self):
                                 tracks=[{'id': int(t.getId()), 'title': to_unicode(t.getTitle())}
                                         for t in self._conf.getTrackList()])
         return jsonify_form(form)
+
+
+class RHEditTrack(RHManageTracksBase):
+
+    def _checkParams(self, params):
+        RHConferenceModifBase._checkParams(self, params)
+        self.track = Track.find_one(id=request.view_args['id'])
+
+    def _process(self):
+        form = TrackForm(obj=FormDefaults(self.track))
```

Comment:
`obj=self.track` - no need to wrap it in FormDefaults if you just pass-through to an object without any of the extra FormDefaults functionality